todoapp/projects/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from users.serializers import UserTodosStatsSerializer
+from projects.models import Project
```

Comment:
1. Spacing group import groups
https://peps.python.org/pep-0008/#imports:~:text=You%20should%20put%20a%20blank%20line%20between%20each%20group%20of%20imports.
2. Order of imports

Also, we prefer aliased imports (especially for local imports), explore why.
```suggestion
from rest_framework import serializers as rest_serializers

from projects import models as projects_models
from users import serializers as users_serializers
```